code/game/objects/items/devices/flashlight.dm

```diff
@@ -36,6 +36,12 @@
 		A.UpdateButtonIcon()
 	return 1
 
+/obj/item/device/flashlight/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] is putting the [src] close to [user.p_their()] eyes and turning it on! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	return (FIRELOSS)

```

Comment:
it's tradition for ```suicide_act()```s... for some reason, it does no harm so either remove them all, or just leave it to be consistent.